### PR TITLE
Design version

### PR DESCRIPTION
**Pull request by vesetok**

### CODE REVIEW

File: frontend/src/Components/MainContainer/MainContainer.module.css [hunk 1]
```diff
@@ -1,37 +1,111 @@
...
+
 .mainContainer {
   display: flex;
   flex-direction: column;
   align-content: center;
   flex-wrap: wrap;
+  background-image: url('file:///D:/Jeopardy%20-%20Fdipardy/App/Fdipardy/frontend/src/images/space-galaxy-background-free-vector.webp');
+  background-size: cover;
+  background-position: center;
+  height: 754px;
```
Comment (LenkoIvanov) [inline frontend/src/Components/MainContainer/MainContainer.module.css:16]: Why 754px? For a more responsive design we need to use rem. Also have you tested out zooming in and out of the page - wheter this breaks the design or not?

File: frontend/src/Components/MainContainer/MainContainer.module.css [hunk 2]
```diff
@@ -1,37 +1,111 @@
...
   flex-direction: column;
   align-items: center;
+  font-size: 30px;
+  color: #c793c7;
+  margin-right: 32px;
+  margin-top: 0px;
+  text-shadow: 6px 4px 22px  #d1ba68;
+  letter-spacing: normal;
+  animation: flip 5s infinite;
+  animation-delay: calc(.15s * var(--i))
```
Comment (LenkoIvanov) [inline frontend/src/Components/MainContainer/MainContainer.module.css:36]: Can you explain this calculation for clarity?

File: frontend/src/Components/MainContainer/MainContainer.module.css [hunk 3]
```diff
@@ -1,37 +1,111 @@
...
+}
+
+@keyframes flip {
+  0%,80% {
```
Comment (LenkoIvanov) [inline frontend/src/Components/MainContainer/MainContainer.module.css:40]: Learned something new here, I haven'r encountered keyframes before. From what I understand the percentages indicate the start (0% ) and finish(100%) of an animation's transition. Can you explain to me what 0%, 80% here means.  When exactly does the rotation end here?

**Review by LenkoIvanov — CHANGES_REQUESTED** [review]
Overall looks pretty good. I have also learned a few things, which is cool! Answer my  questions and attempt to break the web page in some way, before we merge.

File: frontend/src/Components/MainContainer/MainContainer.module.css (hunk 1 above)
Comment (LenkoIvanov) [reply frontend/src/Components/MainContainer/MainContainer.module.css:16]: Shelved for laters

**Comment by vesetok** [comment]
> Learned something new here, I haven'r encountered keyframes before. From what I understand the percentages indicate the start (0% ) and finish(100%) of an animation's transition. Can you explain to me what 0%, 80% here means.  When exactly does the rotation end here? 

The @keyframes CSS at-rule controls the intermediate steps in a CSS animation sequence by defining styles for keyframes (or waypoints) along the animation sequence. This gives more control over the intermediate steps of the animation sequence than transitions.

from
A starting offset of 0%.
to
An ending offset of 100%.

To use keyframes, create a @keyframes rule with a name that is then used by the animation-name property to match an animation to its keyframe declaration. Each @keyframes rule contains a style list of keyframe selectors, which specify percentages along the animation when the keyframe occurs, and a block containing the styles for that keyframe.

File: frontend/src/Components/MainContainer/MainContainer.module.css (hunk 3 above)
Comment (LenkoIvanov) [reply frontend/src/Components/MainContainer/MainContainer.module.css:40]: @vesetok you should reply in the relevant conversation. I have asked what exactly does 0%,80%, as in I have asked for a bit of clarification when exactly will the rotation happen. I don't need the MDN docs clarification.

Comment (vesetok) [reply frontend/src/Components/MainContainer/MainContainer.module.css:40]: @LenkoIvanov These percentages refer to the state of the animation, for example the start state of the animation is written as 0% and the end state as 100%

**Review by LenkoIvanov17 — APPROVED** [review]

**Comment by LenkoIvanov** [comment]
I will approve the pull request.  I have a small note - we are currently not enforcing a commit policy, but your commit messages are sometimes unclear. The ones where you played around with media queries are very undescriptive. A suggestion - next time you write media queries your commit messages could look something like this:
feat: added a media query for resizing panel A
etc

**Pull request merged**